--- Capacitor.podspec	2022-10-25 12:05:24.000000000 -0500
+++ NativeCapacitor.podspec	2022-10-25 12:07:52.000000000 -0500
@@ -10,9 +10,9 @@
   s.ios.deployment_target = '13.0'
   s.authors = { 'Ionic Team' => 'hi@ionicframework.com' }
   s.source = { :git => 'https://github.com/ionic-team/capacitor.git', :tag => package['version'] }
-  s.source_files = 'Capacitor/Capacitor/*.{swift,h,m}', 'Capacitor/Capacitor/Plugins/*.{swift,h,m}', 'Capacitor/Capacitor/Plugins/**/*.{swift,h,m}'
-  s.module_map = 'Capacitor/Capacitor/Capacitor.modulemap'
-  s.resources = ['Capacitor/Capacitor/assets/native-bridge.js']
+  s.source_files = 'ios/Capacitor/Capacitor/*.{swift,h,m}', 'ios/Capacitor/Capacitor/Plugins/*.{swift,h,m}', 'ios/Capacitor/Capacitor/Plugins/**/*.{swift,h,m}'
+  s.module_map = 'ios/Capacitor/Capacitor/Capacitor.modulemap'
+  s.resources = ['ios/Capacitor/Capacitor/assets/native-bridge.js']
   s.dependency 'CapacitorCordova'
   s.swift_version = '5.1'
 end
